offline/ci.sh

```diff
@@ -94,9 +92,10 @@ export HELM_HOME
 
 helm repo add wire https://s3-eu-west-1.amazonaws.com/public.wire.com/charts
 helm repo update
+helm repo add wire-develop https://s3-eu-west-1.amazonaws.com/public.wire.com/charts-develop
 
 # wire_version=$(helm show chart wire/wire-server | yq -r .version)
-wire_version="2.124.0"
+wire_version="2.125.2"
```

Comment:
I guess this has to be changed with every release: How about moving it (and the line above) to near the top of this file and add a comment? :thinking: